src/scenes/levelend-scene.ts

```diff
@@ -84,131 +71,81 @@ export class LevelEndScene {
     this.currentLevel = currentLevel;
-    this.images = {
-      backgroundImg: WIN_BG,
-      star1Img: PIN_STAR_1,
-      star2Img: PIN_STAR_2,
-      star3Img: PIN_STAR_3,
-      winBackgroundImg: DEFAULT_BACKGROUND_1,
-    };
-    loadImages(this.images, (images) => {
-      this.loadedImages = Object.assign({}, images);
-      this.imagesLoaded = true;
-      this.starAnimation();
-    });
-    this.addEventListener();
-    this.audioPlayer = new AudioPlayer();
-    this.setupBg();
+
     this.isLastLevel =
       this.currentLevel !==
         this.data.levels[this.data.levels.length - 1].levelMeta.levelNumber &&
       this.starCount >= 2;
-      this.monster = new Monster(
+    
+    this.monster = new Monster(
       this.canvas,
       monsterPhaseNumber,
-      () => {
-        //Temp fix - wrapping this in a function to avoid reference undefined of 'this'.
-        //To do - hook level-end-scene properly to game state service.
-        this.switchToReactionAnimation
-      }
+      this.switchToReactionAnimation
     );
+    this.showLevelEndScreen();  // Display the level end screen
+    this.addEventListener();
+    this.renderStarsHTML();
   }
 
-  private setupBg = async () => {
-    const { BG_GROUP_IMGS, draw } = loadDynamicBgAssets(
-      this.currentLevel,
-      BACKGROUND_ASSET_LIST
-    );
-    this.background = await createBackground(
-      this.context,
-      this.width,
-      this.height,
-      BG_GROUP_IMGS,
-      draw
-    );
-  };
+  showLevelEndScreen() {
+    
+
+    // Make the levelEnd element visible by setting display to block
+    if (this.levelEndElement) {
+      this.levelEndElement.style.display = 'block';
+    }
+
+    // Render the stars dynamically based on the star count
+    this.renderStarsHTML();
+  }
 
   switchToReactionAnimation = () => {
     if (this.starCount <= 1) {
       if (isDocumentVisible()) {
         this.audioPlayer.playAudio(AUDIO_LEVEL_LOSE);
       }
-      this.monster.changeToSpitAnimation();
+      // this.monster.changeToSpitAnimation();
     } else {
       if (isDocumentVisible()) {
         this.audioPlayer.playAudio(AUDIO_LEVEL_WIN);
         this.audioPlayer.playAudio(AUDIO_INTRO);
       }
-      this.monster.changeToEatAnimation();
+      // this.monster.changeToEatAnimation();
     }
   };
+
   draw(deltaTime: number) {
-    this.background?.draw();
-    if (this.imagesLoaded) {
-      this.context.drawImage(
-        this.loadedImages.backgroundImg,
-        0,
-        0,
-        this.width,
-        this.height + this.height * 0.12
-      );
-      this.drawStars();
+    // No background drawing here as it's handled in HTML
 
-      // this.monster.update(deltaTime);
-      this.closeButton.draw();
-      this.retryButton.draw();
-      if (this.isLastLevel) {
-        this.nextButton.draw();
-      }
+    // Draw monster and buttons
+    this.closeButton.draw();
+    this.retryButton.draw();
+    if (this.isLastLevel) {
+      this.nextButton.draw();
```

Comment:
Optional: This is only a suggestion but I think it would be much cleaner if we could write it like `this.isLastLevel && this.nextButton.draw()`, instead of using IF condition.